docs/containers-auth.json.5.md

```diff
@@ -7,7 +7,13 @@ containers-auth.json - syntax for the registry authentication file
 
 A credentials file in JSON format used to authenticate against container image registries.
 On Linux it is stored at `${XDG_RUNTIME_DIR}/containers/auth.json`;
```

Comment:
Maybe
```suggestion
The primary (read/write) file is stored at `${XDG_RUNTIME_DIR}/containers/auth.json` on Linux;
```
and then refer to “primary” in the last paragraph, instead of saying `XDG_RUNTIME_DIR` on Linux or … on other OS, again?

WRT reverting to explicitly use XDG_CONFIG_HOME… fine, it is equivalent to `nonLinuxAuthFilePath` so it does not introduce new behavior on Windows/macOS; works for me.